book/src/running-validator/validator-stake.md

```diff
@@ -13,7 +13,7 @@ solana-keygen new -o ~/validator-stake-keypair.json
 and use the cli's `create-stake-account` and `delegate-stake` commands to stake your validator with 4242 lamports:
 
 ```bash
-solana create-stake-account ~/validator-stake-keypair.json 4242 lamports
+solana create-stake-account --stake-account ~/validator-stake-keypair.json --amount 4242 --unit lamports
```

Comment:
I rather like the positional parameters.   I find the extra verbosity of `--stake-account`, `--amount`, `--unit` unwelcoming

Reply:
Ugh, me too. Only other approach I can think of is to use a magic keyword for the stake account positional parameter. Thoughts?

Reply:
What's the problem exactly?

Reply:
Using `--ask-seed-phrase stake-account` would mean you wouldn't be passing a keypair path as the first positional arg. `clap` doesn't support variations of positional args from what I can tell.

We would want to support these two methods I think:

```
solana create-stake-account ~/validator-stake-keypair.json 4242 lamports`
```

Secure input:

```
solana create-stake-account 4242 lamports --ask-seed-phrase stake-account
```

Reply:
So, another approach is using what @t-nelson originally suggested:

```
solana create-stake-account ask-seed-phrase 4242 lamports
```

Reply:
Got it, thanks. @t-nelson was right with:
```
$ solana create-stake-account ask-seed 4242 lamports
```

Reply:
Cool, will use that approach then!

Reply:
maybe with more yelling:
```
$ solana create-stake-account ASK 4242 lamports
```

Reply:
I dig it